pkg/ui/react-app/src/pages/graph/Panel.tsx

```diff
@@ -333,6 +344,14 @@ class Panel extends Component<PanelProps & PathPrefixProps, PanelState> {
   };
   handleChangeForceTracing = (event: React.ChangeEvent<HTMLInputElement>): void => {
     this.setOptions({ forceTracing: event.target.checked });
+
+    if (event.target.checked && this.state.traceId) {
```

Comment:
We can display trace ID even if force tracing is not checked...
As long as the response contains the trace ID